app/jobs/socure_docv_results_job.rb

```diff
@@ -6,19 +6,31 @@ class SocureDocvResultsJob < ApplicationJob
   attr_reader :document_capture_session_uuid
 
   # @param [String] document_capture_session_uuid
-  def perform(document_capture_session_uuid:)
+  def perform(document_capture_session_uuid:, async: false)
     @document_capture_session_uuid = document_capture_session_uuid
+    @async = async
```

Comment:
nit: how about adding `async` to attr_reader and then we can reference it as `async`?